app/views/drawings/_drawing.html.haml

```diff
@@ -55,4 +55,11 @@
             .drawing-row
               %strong Context / Story:
               = drawing.story
-            
+            %br
+            .drawing-row
+              - if drawing.image_consent == true
+                %span.fa.fa-check
+              - else
+                %span.fa.fa-times
+              %em
+                Consent given by artist to share image on open source data platforms as deemed appropriate
```

Comment:
Just realised we have this specified twice so we could move it into the drawing_decorator and call it something like "consent_description" to DRY it up